Change the ProgId for Microsoft Edge on Windows

stevenChuCP opened the pull request:
The registry value of Microsoft Edge on my PC(Windows 10 21H1) is `MSEdge.HTM`.
The original code will return **unknown** when my default browser was set as Microsoft Edge.

It's my first pull request. Please inform me if there's any improvement.
Thanks!

Commit:
Change the ProgId for Microsoft Edge on Windows

Change the ProgId of Microsoft Edge to MSEdge.HTM

jakub-g commented:
Hi @stevenChuCP, thanks for the PR! 👍 

I don't have access to a Windows machine in the next few days to test this.
If I don't reply, please remind me about this PR somewhere beginning January.

I also need to think a bit whether to do some special handling of Edge Spartan vs Edge Chromium.

Commit:
Fix: ignore "ProgId" case when querying

Ignore the case of "ProgId".
Some machine has key name as "ProgID" which will cause findstr to not
catch the key value.

eight04 commented:
On my machine, `ProgId` also changed to `ProgID`, which requires addding `/i` flag to `findstr`.

jakub-g commented:
@stevenChuCP thx for the PR - released as 0.5.2! Sorry it took so long.
